RF24.h

```diff
@@ -598,6 +598,46 @@ class RF24 {
      * a `1` means the feature is enabled.
      */
     void printPrettyDetails(void);
+    
+    /**
+     * Put a giant block of debugging information in a char array. This function
+     * differs from printDetails() because it makes the information more
+     * understandable without having to look up the datasheet or convert
+     * hexadecimal to binary. Only use this function if your application can
+     * spare extra bytes of memory.
+     *
+     * @warning use a buffer of sufficient size for debugging_information, start 
+     * with a sizeof >= 1024
+     ^ @param debugging_information The c-string buffer that the debugging 
+     *  information is stored to.
+     * 
+     * @note http://www.cplusplus.com/reference/cstdio/sprintf/
+     ^ sprintf_P formats a C-string in the exact same way as printf_P and outputs 
```

Comment:
Another line that should begin with `*` not `^`. I'm curious if this is actually a typo or are you using a different keyboard (non-querty)?

Reply:
I think it's brackets, which is what I'm using to edit these files with.  I'll pay close attention so I don't keep breaking things.

Reply:
I highly recommend using vscode when/if you can. Even gitpod.io has started using vscode as their online IDE.

Reply:
I will go check that out right now!